pubsub/azuresb/azuresb.go

```diff
@@ -449,13 +449,14 @@ func (s *subscription) ReceiveBatch(ctx context.Context, maxMessages int) ([]*dr
 	defer cancel()
 	var messages []*driver.Message
 
-	// Loop until rctx is Done, or until we've received maxMessages.
-	for len(messages) < maxMessages && rctx.Err() == nil {
+	// Loop until ctx is Done, or until we've received maxMessages.
+	for len(messages) < maxMessages && ctx.Err() == nil {
```

Comment:
I don't think we want this change, as it changes the behavior of ReceiveBatch from waiting at most listenerTimeout (1 second) to waiting until ctx is Done.